### ISSUE TITLE
Error occurred while enabling eZProtector v4.4.2 (Is it up to date?)

### ISSUE THREAD

**Issue by Fabrimat** [DoNotSpamPls/eZProtector#33]
```
[09:36:00] [Server thread/INFO]: [eZProtector] Enabling eZProtector v4.4.2
[09:36:00] [Server thread/ERROR]: Error occurred while enabling eZProtector v4.4.2 (Is it up to date?)
org.bukkit.plugin.messaging.ReservedChannelException: Attempted to register for a reserved channel name ('minecraft:brand')
	at org.bukkit.plugin.messaging.StandardMessenger.registerIncomingPluginChannel(StandardMessenger.java:211) ~[server.jar:git-Spigot-f6a273b-a639ae4]
	at com.github.donotspampls.ezprotector.Main.onEnable(Main.java:107) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[server.jar:git-Spigot-f6a273b-a639ae4]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:339) [server.jar:git-Spigot-f6a273b-a639ae4]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403) [server.jar:git-Spigot-f6a273b-a639ae4]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:431) [server.jar:git-Spigot-f6a273b-a639ae4]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:345) [server.jar:git-Spigot-f6a273b-a639ae4]
	at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:584) [server.jar:git-Spigot-f6a273b-a639ae4]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:546) [server.jar:git-Spigot-f6a273b-a639ae4]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:424) [server.jar:git-Spigot-f6a273b-a639ae4]
	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:294) [server.jar:git-Spigot-f6a273b-a639ae4]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:702) [server.jar:git-Spigot-f6a273b-a639ae4]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
```

Spigot version: 1.13.1

**Comment by DoNotSpamPls** [comment]
Upgrade your Spigot server to 1.13.2, the bug is fixed there.

**Comment by Fabrimat** [comment]
I can't update to 1.13.2 yet for other reasons, so 1.13.1 is officially not supported?

**Comment by DoNotSpamPls** [comment]
Yes, it's not supported, due to problems in Spigot itself that were fixed in 1.13.2

1.13.2 should be 100% compatible with everything 1.13.1 though.
